docs/MigrationGuide.md

```diff
@@ -0,0 +1,118 @@
+# Migration Guide from unleash-android-proxy-sdk
+
+This guide provides detailed steps for migrating your project from the Unleash Android Proxy SDK version to the newer Unleash Android SDK.
+
+We will focus on the [previous sample application](https://github.com/Unleash/unleash-android-proxy-sdk/tree/main/samples/android) and specifically highlighting the changes from this pull request: https://github.com/Unleash/unleash-android-proxy-sdk/pull/83
```

Comment:
Not really sure what you mean here by "highlighting the changes". Is it that:
- we've already got a sample application for the old proxy sdk
- the pull request makes some changes to make it compatible with the new sdk?

```suggestion
We will focus on the [previous sample application](https://github.com/Unleash/unleash-android-proxy-sdk/tree/main/samples/android), specifically highlighting the changes from this pull request: https://github.com/Unleash/unleash-android-proxy-sdk/pull/83
```